packages/mongodb-rag-core/src/eval/getConversationEvalCasesFromCSV.ts

```diff
@@ -0,0 +1,38 @@
+import { parse } from "csv";
+import fs from "fs";
+import { ConversationEvalCase } from "./getConversationEvalCasesFromYaml";
+
+export const getConversationEvalCasesFromCSV = async (
+  csvFilePath: string
+): Promise<ConversationEvalCase[]> => {
```

Comment:
pls add option to include optional transform, for things like adding tags

```suggestion
export type TransformEvalCase = (evalCase: ConversationEvalCase) => Promise<ConversationEvalCase>


export const getConversationEvalCasesFromCSV = async (
  csvFilePath: string,
  tranform?: TransformEvalCase
): Promise<ConversationEvalCase[]> => {
```

Reply:
example `TransformEvalCases` function:

```ts
function addWebDataSourceTag(evalCases){

  return evalCases.map(case => {
    ...case,
    tags: !case.tags?.includes("web) ? [...case.tags, "web"] : case.tags
  })
}
```

Reply:
add something like the above for the web data source eval cases